packages/compass-settings/src/components/sidebar.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { css, cx, spacing, uiColors } from '@mongodb-js/compass-components';
+
+const itemStyles = css({
+  padding: spacing[2],
+  borderRadius: spacing[1],
+  cursor: 'pointer',
+  marginBottom: spacing[1],
+});
+
+const itemHoverStyles = css({
+  '&:hover': {
+    backgroundColor: uiColors.green.light2,
+  },
+});
+
+const itemActiveStyles = css({
+  backgroundColor: uiColors.green.light3,
+});
+
+type SidebarProps = {
+  activeItem: string;
+  onSelectItem: (item: string) => void;
+  items: string[];
+};
+
+const SettingsSideNav: React.FunctionComponent<SidebarProps> = ({
+  activeItem,
+  items,
+  onSelectItem,
+}) => {
+  return (
+    <nav data-testid="settings-sidebar">
+      <ul role="menu">
```

Comment:
Menus are widgets that usually provide access to the list of possible actions, sometimes they include navigation, but in most cases this is a full page one, not a localized one. This side navigation is a tab group, you select a tab and a controlled content is displayed in the main area. Here are some references to related roles: [`tablist`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role), [`tabpanel`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role), [`tab`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role)